app/controllers/test/components/student/concepts_map_controller.rb

```diff
@@ -0,0 +1,4 @@
+class
+  Test::Components::Student::ConceptsMapController < ApplicationController
```

Comment:
My head can't cope with rubocop doing this. 

```suggestion
class Test::Components::Student::ConceptsMapController < ApplicationController # rubocop:disable Layout/LineLength
```